docs/source/markdown/podman-build.1.md

```diff
@@ -394,7 +394,7 @@ The format of `LIMIT` is `<number>[<unit>]`. Unit can be `b` (bytes),
 `k` (kilobytes), `m` (megabytes), or `g` (gigabytes). If you don't specify a
 unit, `b` is used. Set LIMIT to `-1` to enable unlimited swap.
 
-#### **--net**, **--network**=*string*
+#### **--network**=*mode*
```

Comment:
podman build doesn't support the --net alias I opend https://github.com/containers/podman/issues/8322

Reply:
That is a bug, but I am fine with fixing the man pages for now.